usaspending_api/disaster/v2/views/award/amount.py

```diff
@@ -63,7 +63,7 @@ def _file_c_aggregation(self):
             self.is_in_provided_def_codes,
         ]
 
-        group_by_annotations = {"award_identifier": self.unique_file_c_awards}
+        group_by_annotations = {"award_identifier": F("distinct_award_key")}
```

Comment:
Will we ever need the old behavior for `unique_file_c_awards`? If not, we should replace the functionality in that function in order to make this behavior reusable, and prevent misuse of the old algorithm.

Reply:
nevermind, I see the function was deleted altogether